testing/tests/DevExpress.ui.widgets.scheduler/integration.resources.tests.js

```diff
@@ -428,3 +428,86 @@ QUnit.test("Appointment should have correct color after resources option changin
     var $appointments = this.instance.$element().find(".dx-scheduler-appointment");
     assert.equal(new Color($appointments.eq(0).css("backgroundColor")).toHex(), "#ff0000", "Color is OK");
 });
+
+QUnit.module("Integration: Multiple resources", {
+    beforeEach: function() {
+        $("#qunit-fixture").css({ top: 0, left: 0 });
+        this.createInstance = function(options) {
+            this.instance = $("#scheduler").dxScheduler(options).dxScheduler("instance");
+        };
+        $("#qunit-fixture").html(
+            '<div style="width: 1000px; height: 500px;"><div id="scheduler" style="height: 100%;">\
+                <div data-options="dxTemplate: { name: \'template\' }">Task Template</div>\
+                </div></div>');
+    }
+});
+
+QUnit.test("Scheduler with multiple resources and fixed height container has visible horizontal scrollbar (T716993)", function(assert) {
+    this.createInstance({
+        groups: ['facilityId'],
+        crossScrollingEnabled: true,
+        dataSource: [],
+        resources: [{
+            dataSource: [
```

Comment:
Why need many data for testing?

Reply:
It is needed to ensure multiple resources rendering with a horizontal scroll